### ISSUE TITLE
[Accessibility][ Sensory Characteristics - Flow Chart]: Tooltips are not defined for ‘+’ and other controls.

### ISSUE THREAD

**Issue by awalia13** [microsoft/BotFramework-Composer#2006]
**User Experience** 
Users having hearing impairment will get difficulties to know the use of the given controls as no text Sensory Characteristics is given for the controls present in calendar.

**Test Environment**
Browser: Microsoft Edge Canary Version: 82.0.425.0 (Official build)
OS build: 2004 (19564.1005)
Screen Reader: Narrator
URL: [Bot Framework Composer](http://localhost:3000/dialogs/Main)
[**Prerequisite Here**](https://github.com/microsoft/BotFramework-Composer)

**Actual Result**
Tooltip of ‘+’ control is not defined.

**Expected:**
Tooltip’s should be defined for the controls which doesn’t have text associated with them.

**Repro Steps:**
1. Open the above mentioned URL in EDGE.
2. Select 'Design Flow' from left navigation.
3. Now, navigate to '+' button with tab in the flow chart.
4. Observe the issue.

**Note:**
Same issue observed throughout the website, tooltips are not defined for any of the controls.

**MAS Impacted:** [MAS1.3.3]( https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={1823da5a-e8fb-4e63-b881-7eccb312c358})

**Attachment:**
[tooltips are not defined for the + button.zip](https://github.com/microsoft/BotFramework-Composer/files/4225289/tooltips.are.not.defined.for.the.%2B.button.zip)

**Comment by DesignPolice** [comment]
Okay I will look at how we could add a tooltip to this...

**Comment by corinagum** [comment]
![tooltips are not defined for the + button](https://user-images.githubusercontent.com/14900841/75415830-95a92e80-58e1-11ea-95b9-8a99c9a59d28.png)

.png file uploaded here for ease of access, but note that the .zip file also contains a video

**Comment by DesignPolice** [comment]
@corinagum on this issue... I chatted with Marieke and she doesn't want to add anything to this button... as far as a (?) tool tip or ellipsis menu - 

It is just a button, not a node - so it would be the same as other button.  Thoughts?

thanks m

**Comment by corinagum** [comment]
Unfortunately the tooltip is required, but it's not necessary for design to create a custom tooltip if not desired. It looks like any button that does not have text requires a text description on hover.

I don't have access, but if you need more details you can take a look at the MAS.1.3.3 link above... but I don't think you'll have access either. :D

Here's an external link: https://inclusive-components.design/tooltips-toggletips/

We had this come up for the Emulator audit as well. Below is the hover state for the close tab:

![image](https://user-images.githubusercontent.com/14900841/75922457-e526b780-5e17-11ea-9f62-29ae560fa424.png)
This is the default design (in this case, I think from Windows). There is no custom tooltip component for the Emulator.

If design doesn't want to add the tooltip, that shouldn't be a problem; the dev can use the default tooltip styling by adding the tooltip (e.g. `tooltip="Add new node"`) attribute to the button.

**Comment by DesignPolice** [comment]
Oh @corinagum, my bad! I wasn't understanding - We should use the same tool tips that we have on ?'s in the Right Pane.

<img width="343" alt="Screen Shot 2020-03-04 at 1 28 57 PM" src="https://user-images.githubusercontent.com/45223620/75924863-87489e80-5e1c-11ea-93f3-06f054898a3b.png">

If that gets lost we can use a blue version in Fluent Web.

![Screen Shot 2020-03-04 at 1 30 33 PM](https://user-images.githubusercontent.com/45223620/75924910-a1827c80-5e1c-11ea-961b-53aec3a265d1.png)

**Comment by corinagum** [comment]
Ok, I'll see if the audit team filed any issues against the tooltip component you showed above. Assuming there aren't any, there shouldn't be any problem for accessibility in using this component. :)

**Comment by Amit8527510735** [comment]
#Nonbenchmark;

**Comment by taicchoumsft** [comment]
@beyackle , does this fall under your purview?

**Comment by taicchoumsft** [comment]
Hi, @Amit8527510735 , this was fixed back in June 2020 and my understanding is this is potentially awaiting your review/signoff before we can close the ticket.  Is this still the right course of action?

**Comment by benbrown** [comment]
@Amit8527510735 Can you review and close this?
